packages/cli/bin/backstage-cli

```diff
@@ -5,12 +5,60 @@ const path = require('path');
 // Figure out whether we're running inside the backstage repo or as an installed dependency
```

Comment:
Add the Copyright `notice` in this file too?

Reply:
Yup :p

Reply:
#494